### PR TITLE
Use dotnet-20-runtime-rhel7 image

### PR DESCRIPTION
**Pull request by tmds**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,5 @@
-FROM registry.access.redhat.com/rhel7
+FROM registry.access.redhat.com/dotnet20-runtime-rhel7
```
Comment (jerboaa) [inline Dockerfile]: The name will be `dotnet_beta/dotnet-20-runtime-rhel7`, then for GA will change to `dotnet/dotnet-20-runtime-rhel7`. I suggest you use:
`registry.access.redhat.com/dotnet_beta/dotnet-20-runtime-rhel7`

File: Dockerfile [hunk 2]
```diff
@@ -1,12 +1,5 @@
...
+ADD bin/Release/netcoreapp2.0/rhel.7-x64/publish/. .
 
-ADD bin/Release/netcoreapp2.0/rhel.7.3-x64/publish/. /opt/app-root/src/
-
-WORKDIR /opt/app-root/src/
-
-EXPOSE 5000 
-
-CMD ["/bin/bash", "-c", "/opt/app-root/src/couchbase_demo"]
+CMD ["scl", "enable", "rh-dotnet20", "--", "dotnet", "couchbase_demo.dll"]
```
Comment (jerboaa) [inline Dockerfile]: Hmm, the runtime image should have the collection enabled by default. Thus, `dotnet couchbase_demo.dll` should work too. I'm going to test this to verify.

Comment (tmds) [inline Dockerfile]: That's also how I'd like it to work. That doesn't work today.

File: Dockerfile (hunk 1 above)
Comment (tmds) [inline Dockerfile]: Didn't know if I could put this in a public PR...

File: Dockerfile (hunk 2 above)
Comment (jerboaa) [inline Dockerfile]: OK. I'll file a bug.

File: Dockerfile (hunk 1 above)
Comment (jerboaa) [inline Dockerfile]: The names are no secrets really.

**Comment by tmds** [comment]
Cleaning up my open PRs.

**Pull request closed without merging**